packages/generators/src/handlers/default.ts

```diff
@@ -82,6 +82,14 @@ export async function questions(self: CustomGenerator, Question: Record<string,
         self.force,
     );
 
+    const { isExtractPlugin } = await Question.Confirm(
+        self,
+        'isExtractPlugin',
+        'Do you want to extract CSS for every file?',
```

Comment:
This is misleading for beginners, and we should phrase this differently. Maybe, `Do you want to split CSS into seperate files`?

Reply:
we don't do split here, we extract :smile: